src/internal/Subject.ts

```diff
@@ -91,6 +91,10 @@ export class Subject<T> extends Observable<T> implements SubscriptionLike {
     this.observers = null!;
   }
 
+  isObeserved() {
+    return this.observers?.length > 0;
+  }
+
```

Comment:
given we have a property `closed`, I think it's more inline with other parts of RxJS to not prefix with `is` and to use a property. Also `Obeserved` is a typo. :)

```ts
get observed() {
  return 0 < this.observers?.length;
}
```

Reply:
fixed